submissions/another96/dom-task/myScript.js

```diff
@@ -0,0 +1,59 @@
+  },
+  {
+    id: "sve",
+    image: "icons/svelve.png",
+    alt: "svelve",
+    description: "Svelte is a free and open-source front end compiler created by Rich Harris and maintained by the Svelte core team members. Svelte is not a monolithic JavaScript library imported by applications: instead, Svelte compiles HTML templates to specialized code that manipulates the DOM directly, which may reduce the size of transferred files and give better client performance; application code is also processed by the compiler, inserting calls to automatically recompute data and re-render UI elements when the data they depend on is modified. This also avoids the overhead associated with runtime intermediate representations, such as virtual DOM, unlike traditional frameworks (such as React and Vue) which carry out the bulk of their work at runtime, i.e in the browser. The compiler itself is written in TypeScript. Its source code is licensed under MIT License and hosted on GitHub."
+  },
+  {
+    id: "jqe",
+    image: "icons/jquery.png",
+    alt: "jquery",
+    description: "Svelte is a free and open-source front end compiler created by Rich Harris and maintained by the Svelte core team members. Svelte is not a monolithic JavaScript library imported by applications: instead, Svelte compiles HTML templates to specialized code that manipulates the DOM directly, which may reduce the size of transferred files and give better client performance; application code is also processed by the compiler, inserting calls to automatically recompute data and re-render UI elements when the data they depend on is modified. This also avoids the overhead associated with runtime intermediate representations, such as virtual DOM, unlike traditional frameworks (such as React and Vue) which carry out the bulk of their work at runtime, i.e in the browser. The compiler itself is written in TypeScript. Its source code is licensed under MIT License and hosted on GitHub."
+  }
+];
+
+let selectedFramewok = document.getElementById("rea");
+var img = document.getElementById("img");
+img.src = contentData[0].image;
+img.alt = contentData[0].alt;
+document.getElementById("text").innerHTML = contentData[0].description;
+var menu = document.querySelector(".menu");
+
+menu.addEventListener("click", ({ target }) => {
+  const value = target.id;
+  if (selectedFramewok) {
+    selectedFramewok.classList.remove("selectedFramework");
+  }
+  selectedFramewok = target;
+  selectedFramewok.classList.add("selectedFramework");
+
+  for (let i in contentData) {
+    if (contentData[i].id == value) {
+      img.src = contentData[i].image;
+      img.alt = contentData[i].alt;
+      document.getElementById("text").innerHTML = contentData[i].description;
```

Comment:
You repeated this code twice. Create a separate function for it.
And try to create HTML elements, not update them.